internal/storage/v2/api/tracestore/reader.go

```diff
@@ -88,6 +88,29 @@ type TraceQueryParams struct {
 	NumTraces     int
 }
 
+// FoundTraceID represents a chunk of trace IDs that match specific query parameters.
+type FoundTraceID struct {
+	// TraceID is a slice of trace IDs that match the query parameters.
```

Comment:
I suggest removing comments from the fields, as the meaning is trivial and the optional nature of timestamps can be called out once in the struct comment.